QtTesterApp/QtTestAppView.cpp

```diff
@@ -76,11 +76,6 @@ QStringList DEBUG_GetAllFiles(const QString& path, const QString& extension)
 // TODO: REMOVE and cleanup 
 void QtMainWindowView::DEBUG_InitRenderView()
 {
-	// TODO: DOESN't WORK, get set to null, as the release version is called, make better debugging code, that doesn't affect the widget, maybe
```

Comment:
This sets the debug FileGetter (stanThis sets the debug FileGetter (stanard-in for the release asset Calback) why is this deleted?

Reply:
The file getter was rewritten so if the static that contains it is a nullptr, it uses the debug code. If it's set, it uses the release code.